webapp/app/(routes)/page.tsx

```diff
@@ -6,10 +6,11 @@ import ThirdPartyServicesKeys from "@/components/third-party-services/third-part
 import FileUpload from "@/components/resume/file-upload/file-upload";
 import ResumeGlance from "@/components/resume/resume-glance/resume-glance";
 import Hero from "@/components/hero/hero";
+import Footer from "../_components/footer/footer";
 
 export default function Home() {
   return (
-    <main>
+    <main className="text-[white]">
```

Comment:
👀  There's no need to wrap the pre-defined colour within square brackets.

Instead, we I suggest we simply leverage the existing baked-in Tailwind CSS class `text-white`, which is also consistently used elsewhere throughout the web app codebase.

```suggestion
    <main className="text-white">
```